### ISSUE TITLE
More bugs and Error messages continue

### ISSUE THREAD

**Issue by ranrinc** [fwestenberg/reolink_dev#240]
**Describe the bug**
error log continues
Received message for unregistered webhook 45616ccf193e7dc4c869882b5fb97f33d5a1dab3fd89c292f4b16854ccbc2d78 from 192.168.xxx.xxx

On current HA 2021.8 after rebooting the live feed show green screen until being refresh

**To Reproduce**
It happens repeatedly

**Expected behavior**
No error

**Screenshots**
![Screen Shot 2021-08-04 at 8 58 20 PM](https://user-images.githubusercontent.com/15801696/128193947-9c0b074d-d2ce-46ed-b081-990ad1c2c77a.png)

**Environment:**
Please provide useful information about your environment, like:
 - Home Assistant version 2021.8
 - Reolink camera model RLC 511W
 - Camera firmware number v3.0.0.142_20121804
 - latest reolink addone 0.19

**Additional context**

**Comment by milandzuris** [comment]
me to

**Comment by Jai5-lang** [comment]
I'm also getting this error.

**Comment by cpainchaud** [comment]
is it still happening today?

thank you

**Comment by ranrinc** [comment]
> is it still happening today?
> 
> thank you

Yes, last week download the latest version and the error still coming in

**Comment by cpainchaud** [comment]
error message "Received message for unregistered webhook..." has likely nothing to do with it. Can you try to run a lower loggin level like warning or debug ?

The API call to get an image runs always the same way so I would suspect a camera bug if no error message is showing after enable extensive logging

**Comment by ranrinc** [comment]
It's definitely coming from this addon, since I delete it the error log stop sending the error message anymore. Connection from the camera using ffmpeg method works and I'm not alone on this issue

**Comment by ranrinc** [comment]
> the error message ""Received message for unregistered webhook..."" is coming from a bad camera unsubscription yes but there is no relationship with your issue of a green picture.
> 
> now about that green screen, does it happens only after reboot and then it vanishes until you reboot again or doest it come back from time to time?

I'm not sure need to retest.
